CgfConverter/Renderers/Wavefront/Wavefront.Material.cs

```diff
@@ -65,73 +65,77 @@ public void WriteMaterial(CryEngine cryEngine)
-                                file.WriteLine("map_Ns {0}", textureFile.ToString());
-                                break;
-
-                            case CryEngineCore.Material.Texture.MapTypeEnum.Custom:
-                                // <Texture Map="Custom" File="objects/spaceships/ships/aegs/textures/interior/metal/aegs_int_metal_painted_red_ddna.tif"/>
-                                // file.WriteLine("decal {0}", textureFile);
-                                break;
-
-                            case CryEngineCore.Material.Texture.MapTypeEnum.BlendDetail:
-                                // <Texture Map="BlendDetail" File="textures/unified_detail/metal/metal_scratches-01_detail.tif">
-                                break;
-
-                            case CryEngineCore.Material.Texture.MapTypeEnum.Opacity:
-                                // <Texture Map="Opacity" File="objects/spaceships/ships/aegs/textures/interior/blend/interior_blnd_a_diff.tif"/>
-                                file.WriteLine("map_d {0}", textureFile.ToString());
-                                break;
-
-                            case CryEngineCore.Material.Texture.MapTypeEnum.Environment:
-                                // <Texture Map="Environment" File="nearest_cubemap" TexType="7"/>
-                                break;
-
-                            default:
-                                break;
+                            StringBuilder textureFile = new StringBuilder(ResolveTexFile(texture.File, Args.DataDir));
+
+                            // TODO: More filehandling here
+
+                            if (this.Args.PngTextures)
+                                textureFile = textureFile.Replace(".dds", ".png");
+                            else if (this.Args.TgaTextures)
+                                textureFile = textureFile.Replace(".dds", ".tga");
+                            else if (this.Args.TiffTextures)
+                                textureFile = textureFile.Replace(".dds", ".tif");
+
+                            switch (texture.Map)
+                            {
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Diffuse:
+                                    file.WriteLine("map_Kd {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Specular:
+                                    file.WriteLine("map_Ks {0}", textureFile.ToString());
+                                    file.WriteLine("map_Ns {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Bumpmap:
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Detail:
+                                    // <Texture Map="Detail" File="textures/unified_detail/metal/metal_scratches_a_detail.tif" />
```

Comment:
Can you remove unused comments?  I know they are left over from when I originally wrote them (bad, I know!), but as long as this file is being modified, might as well clean it up. :)